tree/ntuple/v7/src/RMiniFile.cxx

```diff
@@ -512,9 +513,14 @@ struct RTFFile {
 /// A zero UUID stored at the end of the TFile record
 struct RTFUUID {
    RUInt16BE fVersionClass{1};
-   unsigned char fUUID[16] = {0};
+   unsigned char fUUID[16];
 
-   RTFUUID() = default;
+   RTFUUID()
+   {
+      TUUID uuid;
```

Comment:
TFile has the `kReproducible` flag which, if set, ensures the UUID is always 0. Should we provide the same mechanism for RMiniFile?

Reply:
I don't think we need to add it for the simple file writer. Users who wish to use reproducible files can provide their own `TFile` and the the proper file writer will take care of handling the UUID.